requirements.txt

```
@@ -0,0 +1,6 @@
+asgiref==3.5.2
+certifi @ file:///private/var/folders/nz/j6p8yfhx1mv_0grj5xl4650h0000gp/T/abs_884c889c-96af-444f-bd6d-daddb5e9a462ykj3l5n_/croots/recipe/certifi_1655968814730/work/certifi
+Django==4.0.6
+django-cors-headers==3.13.0
+mysqlclient==2.1.1
+sqlparse==0.4.2
```

```suggestion
Django==4.0.6
django-cors-headers==3.13.0
mysqlclient==2.1.1
```
`pip install django`와 같이 파이썬 패키지를 설치하게되면 관련 패키지에 필요한 다른 패키지도 같이 설치됩니다. 그렇기 때문에 단순히 `pip freeze > requirements.txt` 명령어를 사용해서 `requirements.txt` 파일을 생성할 경우에 정확히 어떤 패키지를 설치했는지 알 수 없습니다. 그렇기 때문에 아래와 같이 설치한 패키지와 버전만을 `requirements.txt` 파일에 추가해주셔야 합니다.